tests/tck/features/ttl/TTL.feature

```diff
@@ -112,7 +112,7 @@ Feature: TTLTest
     Then the execution should be successful
     When executing query:
       """
-      CREATE TAG woman(name string, email string, age int, gender string, row_timestamp timestamp) ttl_duration = 0, ttl_col = "row_timestamp";
+      CREATE TAG woman(name string, email string, age int32, gender string, row_timestamp timestamp) ttl_duration = 0, ttl_col = "row_timestamp";
```

Comment:
Note I change from int to int32, so we will have room to "upgrade" it into `int64` later.